bc.py

```diff
@@ -97,6 +97,18 @@ def add_pa(s, orders):
     r = s.get('https://www.bike-components.de/en/checkout/finalize/')
     doc = lxml.html.document_fromstring(r.text)
 
+    if not doc.cssselect('#voucher__token'):
```

Comment:
The other changes are good, but I need to test this before I can merge.

Reply:
#1